paper-typeahead-input.html

```diff
@@ -158,7 +158,8 @@
        */
       localCandidates:{
         type:Array,
-        value:[]
+        value:[],
+        observer: '_candidatesUpdated'
```

Comment:
I would recommend a more specific name for this function, because it is listening to the variable, localCandidates, so I think _localCandidatesUpdated is better.